### PR TITLE
added docker-compose for cometbft

### PR DESCRIPTION
**Pull request by rwlockbg**

### CODE REVIEW

File: cometbft/init.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+if [ ! -f "/cometbft/config/priv_validator_key.json" ] || [ ! -f "/cometbft/config/node_key.json" ]; then
+    echo "Initializing CometBFT..."
+    cometbft init -k "secp256k1" --home /cometbft
+else
+    echo "CometBFT already initialized, skipping init process..."
+    exec cometbft node \
```
Comment (arminsabouri) [inline cometbft/init.sh]: Should this be if else? It seems like we always want the else block executed

File: cometbft/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.PHONY: build-cometbft-local start-cometbft-local stop-cometbft-local 
+
+build-cometbft-local:
+	docker-compose -f docker-compose-cometbft-local.yml up --build -d
+
+start-cometbft-local:
+	docker-compose -f docker-compose-cometbft-local.yml up -d
+
+stop-cometbft-local:
+	docker-compose  -f docker-compose-cometbft-local.yml down
```
Comment (arminsabouri) [inline cometbft/Makefile]: Could we add one more command for restarting the nodes?
specifically executing the `cometBFT unsafe-reset-all --home X` command

File: cometbft/init.sh (hunk 1 above)
Comment (rwlockbg) [reply cometbft/init.sh]: agreed, change

File: cometbft/Makefile (hunk 2 above)
Comment (rwlockbg) [reply cometbft/Makefile]: yep, added this

**Review by arminsabouri — APPROVED** [review]

**Pull request merged**